### DISCUSSION TITLE
Observations and questions about "Override Accent Blur"

### DISCUSSION THREAD

**Discussion by Skyyblaze** [Maplespe/DWMBlurGlass#287] [General]
I find "Override Accent Blur" is a very exciting feature, thanks for the hard work! I didn't make this a Bug Report as I use many other third-party UI mods and just wanted to share some observations that may or not be helpful for the development of DWMBlurGlass.

First I use MicaForEveryone to enable "Blur Behind" with a transparent Discord skin to have some blur in Discord.

![grafik](https://github.com/Maplespe/DWMBlurGlass/assets/22203339/f2af889a-a378-41c1-a694-6ff91d77cdfd)

![grafik](https://github.com/Maplespe/DWMBlurGlass/assets/22203339/3464a561-ac83-4f8b-9987-2982f9a1c601)

If I enable Override Accent Blur, this effect becomes opaque and loses all transparency:

![grafik](https://github.com/Maplespe/DWMBlurGlass/assets/22203339/c9e79c81-b9b6-4555-af61-1f35409afe52)

However if I set the Effect Type to Aero, some very faint transparency is restored:

![grafik](https://github.com/Maplespe/DWMBlurGlass/assets/22203339/b72f9274-6451-494b-8771-697d8dafbcae)

Furthermore I noticed that Override Accent Blur does nothing for the Windows 10 Start Menu when enabled with ExplorerPatcher:

![grafik](https://github.com/Maplespe/DWMBlurGlass/assets/22203339/7863ece5-4906-40cb-be9d-52f8a055a1bf)

So they might have implemented a custom Blur solution.

And lastly I wanted to ask, would it possible and for interest of the project to be able to set a different effect for the Accent Blur and the "Normal" Blur?

So for example have the Normal Blur set to Mica for Explorer and the Accent Blur set to Aero for example.

I realize the intend of DWMBlurGlass is mostly to recreate the old Vista/Windows 7 Aero DWM and that my customization goes a bit in another direction but I thought I would ask and I hope my observations help the dev team or other people.

Either way thanks for the hard work everyone involved, I really appreciate the project!

**Comment by kfh83** [comment]
> I realize the intend of DWMBlurGlass is mostly to recreate the old Vista/Windows 7 Aero DWM and that my customization goes a bit in another direction but I thought I would ask and I hope my observations help the dev team or other people.
> 

A bit offtopic, but actually, it's intention is (or was originally) to just be an all-in-one solution for titlebar transparency.  All of the Windows 7 and Aero work has been product of countless requests of those who want to do their transformation work, which are a very large majority of DWMBlurGlass users, but your requests should still be valued.

**Reply by Skyyblaze** [reply]
I see, thanks for the correction. It's not surprising to me that many push back to the Vista and Win7 Aero design, I do too in parts although I tend to favor Aero Custom Styles instead of Stock Aero like this one for example: https://www.deviantart.com/minhtrimatrix/art/Luna-170290046

But sadly nobody seems to make these kinds of style anymore, especially not for Windows 10 and 11.

Nonetheless I'll follow DWMBlurGlass' development still eagerly!
